lib/EffectSequence.re

```diff
@@ -0,0 +1,15 @@
+type t = unit => unit;
+
+let noop = () => ();
```

Comment:
I think you wanna use `Sys.opaque_identity` here?

Reply:
Thanks for the tip. I just tried replacing `() => ()` with `opaque_identity` but the results were mixed, no real difference (worked ~0.01% better or worse than `() => ()` depending on a benchmark.)